packages/core/src/html/NodeProcessor.ts

```diff
@@ -112,7 +112,7 @@ export class NodeProcessor {
     let currentFrontMatter = {};
     const frontMatter = cheerio(node);
```

Comment:
```suggestion
    const frontmatter = cheerio(node);
```

As well as other places in this file

Reply:
Hi @tylt! Thanks for your suggestion, I initially decided not to touch the variable names in fear of breaking things. However, after looking at your suggestion I will push a new commit fixing those in code today!